primitives/src/lib.rs

```diff
@@ -1,5 +1,65 @@
-#![no_std]
+#![cfg_attr(not(feature = "std"), no_std)]
 
 use sp_core::U256;
 
 pub type Difficulty = U256;
+
+#[cfg(all(feature = "std", feature = "prometheus"))]
+pub mod metrics {
```

Comment:
Do we have any example how to enable prometheus to consume these metrics? I don't want to assert on them in too much details but at least make sure they work (e.g. don't crash, return some actual data).